exercises/acronym/lib/example.dart

```diff
@@ -0,0 +1,15 @@
+class Acronym {
+  String abbreviate(final String input) {
+    StringBuffer acronymBuilder = new StringBuffer();
+
+    RegExp separatorsExp = new RegExp(r'( )|(-)');
```

Comment:
So now we're only splitting on spaces and hyphens?

Should we split on any kind of white space? I think `\w` matches all whitespace.

We don't need to consider any other non-alphanumeric characters as part of this?

Reply:
I guess that makes a lot of sense based on the test cases. We don't have to split on all non-alphanumeric characters.

Although, I'm curious... If any splitted substring started with a non-alphanumeric character... We could see some really weird acronyms.

Reply:
@jvarness If you can find an acronym that starts with that (phrase, company name, etc), then we can certainly add that as a test case. :)